templates/trusted-node/node-config.toml

```diff
@@ -68,7 +68,7 @@ Environment = "production"
 # log. Generally we'll switch to debug if we want to troubleshoot
 # something specifically otherwise we leave it at info
 # ------------------------------------------------------------------------------
-Level = "info"
+Level = "debug"
```

Comment:
We might need to revert back to `info`?

Reply:
Good catch! Will change it back to info.